static/src/stylesheets/module/content-garnett/_pillars.scss

```diff
@@ -190,15 +207,26 @@
         }
     }
 
-    &.tone-feature--item.rich-link--pillar-sport {
-        @include richLink($sport-garnett-main-1, $garnett-neutral-3);
-    }
-
     &.tone-live--item {
-        @include richLink($live-garnett-main-2, $live-garnett-main-1);
+        &.rich-link--pillar-news {
+            @include richLink(#ffffff, $news-garnett-feature);
+        }
 
-        .u-faux-block-link--hover {
-            background-color: darken($live-garnett-main-1, 5%);
+        &.tone-comment--item,
```

Comment:
Is it possible for an element to have the classes `.tone-live--item` and `.tone-comment--item` simultaneously?

Reply:
Nope, thanks that's now fixed